cdap-docs/admin-manual/source/operations/preferences.rst

```diff
@@ -119,6 +119,14 @@ the argument will be set for all programs that use ``myTable``. In order to set
 program ``myMR``, the dataset argument can be nested inside the program scope as
 ``mapreduce.myMR.dataset.myTable.cache.seconds=30``.
 
+Overriding Logical Start Time with Runtime Arguments
+====================================================
+
+The runtime argument ``logicalStartTime`` can be used to override the logical start time of a Workflow, MapReduce,
```

Comment:
Hm I see your point. Perhaps we should just treat logical start time as a runtime argument instead of treating it specially? We could then say this (and move this above the scoped workflow runtime args section):

```
System Populated Runtime Arguments
==============================

Every program run, CDAP will populate the runtime arguments with pre-defined values:

"logical.start.time" - the start time of the run as a timestamp in milliseconds. If the run was started by a schedule, this will be equal the trigger time for the schedule. For example, if the schedule was set to run at midnight Jan 1 2016 UTC, the logical start time would be 1451606400000.

If any of these arguments are passed in to start a program run, they will override the system populated value.
```

and then in the scoping section, mention that they apply to system populated runtime args as well.